plugins/woocommerce/client/blocks/assets/js/blocks/product-template/style.scss

```diff
@@ -37,15 +37,21 @@ $min-product-width: 150px;
 		@include break-small {
 			@for $i from 2 through 6 {
 				&.is-flex-container.columns-#{ $i } > li {
-					width: calc((100% / #{$i}) - 1.25em + (1.25em / #{$i}));
+					// Use CSS custom property with fallback for dynamic width calculation
+					width: calc((100% / #{$i}) - var(--wc-product-template-gap, 1.25em) + (var(--wc-product-template-gap, 1.25em) / #{$i}));
```

Comment:
Is it worth saving `var(--wc-product-template-gap, 1.25em)` to a variable, just so this is a bit easier to grok?